nexus/db-queries/src/db/datastore/lldp.rs

```diff
@@ -0,0 +1,178 @@
+use super::DataStore;
+use crate::context::OpContext;
+use crate::db;
+use crate::db::error::public_error_from_diesel;
+use crate::db::error::ErrorHandler;
+use crate::db::model::LldpLinkConfig;
+use async_bb8_diesel::AsyncRunQueryDsl;
+use chrono::Utc;
+use diesel::ExpressionMethods;
+use diesel::QueryDsl;
+use diesel::SelectableHelper;
+use ipnetwork::IpNetwork;
+use omicron_common::api::external;
+use omicron_common::api::external::Error;
+use omicron_common::api::external::LookupResult;
+use omicron_common::api::external::Name;
+use omicron_common::api::external::ResourceType;
+use omicron_common::api::external::UpdateResult;
+use uuid::Uuid;
+
+// The LLDP configuration has been defined as a leaf of the switch-port-settings
```

Comment:
Should this whole thing be `//!`? It seems like kind of a doc comment, but I'm unsure.